assets/dev/js/editor/controls/date-time.js

```diff
@@ -10,6 +10,32 @@ export default class extends ControlBaseDataView {
 		this.ui.input.flatpickr( options );
 	}
 
+	onBaseInputChange() {
+		super.onBaseInputChange( ...arguments );
+
+		if ( ! this.model.attributes?.validation ) {
+			return;
+		}
+
+		if ( this.model.attributes?.validation.minDate ) {
+			const controlName = this.model.attributes.validation.minDate.control_name;
+			const startDate = this.options.container.settings.get( controlName );
+
+			if ( ! startDate ) {
+				return;
+			}
+
+			const endDate = this.ui.input.val();
```

Comment:
the current control isn't always the end date, I think we need to add something like `validation.maxDate`

Maybe we can change the API in PHP to indicate that the 2 controls are linked, and then you won't need to specify the validations config there, everything will be validated automatically once the developer linked them. Plus, we probably need to talk with @rami-elementor about it

Reply:
@rami-elementor 
What do you think?